python/sparktk/frame/ops/flatten_columns.py

```diff
@@ -0,0 +1,99 @@
+
+def flatten_columns (self, columns, delimiters=None):
```

Comment:
I'm thinking we should change the API to accept arbitrary tuples...  i.e. pass tuples if you want to change the delimeter.  example:
``  f.flatten_columns("a", ("b", "|"), "c", ("d", ";")) ``

same for unflatten...  follows the spirit of sorted_k, categorical summary...